inbm-lib/inbm_lib/constants.py

```diff
@@ -52,6 +48,9 @@
 QUERY_CMD_CHANNEL = 'dispatcher/query'
 HOST_QUERY_CHANNEL = 'manageability/request/query'
```

Comment:
**<h2>$\color{#FFD700}Coding Style$</h2>**
**Suggestion:** The addition of RESTART_CMD_CHANNEL constant is good for maintainability, but it should be grouped logically with other similar constants. Consider grouping all channel-related constants together for better readability and organization of the constants file.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
# Grouping channel-related constants
QUERY_CMD_CHANNEL = 'dispatcher/query'
HOST_QUERY_CHANNEL = 'manageability/request/query'
RESTART_CMD_CHANNEL = 'manageability/request/restart'
```